SCAN.md

```diff
@@ -5,7 +5,8 @@ Documentation on the Seattle Coronavirus Assessment Network (SCAN) project
 - [Adding a new language](#adding-a-new-language)
 
 ## Return of results portal
-Currently, we return SCAN participant's results via a dedicated page for SCAN at the [UW LabMed SecureLink portal].
+Currently, we return SCAN participant's results via a dedicated page for SCAN at the [UW LabMed SecureLink portal]. (See the private [securelink GitHub repo].)
+
```

Comment:
@kschwabefry is this what you imagined?

Reply:
Yes! And good call to explicitly mention that its private to reduce confusion.